monitor/service/api/v10.py

```diff
@@ -15,6 +15,8 @@
 
 import ConfigParser
 import os
+import filecmp
+import shutil
```

Comment:
you don't seem to be using these libs.

Reply:
Modification made in commit 34fab04e5bd16c1e03a39164fa0c77717ad7cf3d